src/clib/pioc_sc.c

```diff
@@ -34,214 +32,232 @@ typedef long long PIO_Offset;
 #include <pio_internal.h>
 #endif
 
-/**
- ** The default target blocksize for each io task when the box rearranger is used.
- */
-//#define default_blocksize 884736
-#define default_blocksize       1024
-//#define default_blocksize       388816
-/**
- ** The target blocksize for each io task when the box rearranger is used.
- */
+/** The default target blocksize for each io task when the box
+ * rearranger is used. */
+#define default_blocksize 1024
```

Comment:
I don't know. I deleted some commented out values which are higher, as you can see. So probably someone at some point tried to make this smaller and just forgot to make it bigger again. ;-)

Would you like me to use one of the other values? Make this a Cmake configurable option? Start a github issue so it is not forgotten?